src/styles/blocks/header.scss

```diff
@@ -0,0 +1,101 @@
+.header {
+  @include flex(space-between, center);
+
+  background-color: $white-color;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+@media screen and (max-width: 992px) {
+  .header {
+    height: 60px;
+  }
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  line-height: 1;
+
+  @include image-size(40px, 40px);
+}
+
+.header-nav {
```

Comment:
It could be just `nav` and as it's an independent block move it to a separate file